src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -476,6 +478,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	public sendNotebookTelemetryEvent(action: TelemetryKeys.TelemetryAction | TelemetryKeys.NbTelemetryAction, additionalProperties: ITelemetryEventProperties = {}): void {
```

Comment:
Actually thinking more - let's name this `sendNotebookTelemetryActionEvent` to be even more specific of the type of event we're sending